manifests/r/RileyTestut/AltServer/1.6/RileyTestut.AltServer.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=QUSU.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RileyTestut.AltServer
+PackageVersion: "1.6"
```

Comment:
This technically writes `1.6.00` as it's display version. I don't think this will cause any issues, since 1.6 should be equal to 1.6.00